packages/sdk/src/utils/LocalConfigurationDecorator.ts

```diff
@@ -0,0 +1,68 @@
+import { StudioOptionsDeltaUpdate, WellKnownConfigurationKeys } from '../types/ConfigurationTypes';
+import { getEditorResponseData } from './EditorResponseData';
+import { ConfigurationController } from '../controllers/ConfigurationController';
+
+export class LocalConfigurationDecorator {
```

Comment:
Should probably extend `ConfigurationController` or implement a common interface